### ISSUE TITLE
[F.Rq.] Colour field for watch entries to complement notes

### ISSUE THREAD

**Issue by YoshiRulz** [TASEmulators/BizHawk#4050]
see also #4046

**Comment by Morilli** [comment]
Are you suggesting a color selector to make the actual watch in the list be displayed with a certain background color or are you suggesting coloring the fields in the watch editor? #4046 does not mention color at all, so I don't know what this means.

**Comment by YoshiRulz** [comment]
Yes the background. The suggestion comes from [this forum post](https://tasvideos.org/Forum/Posts/531883).
